manager/dispatcher/dispatcher.go

```diff
@@ -213,8 +216,15 @@ func (d *Dispatcher) Run(ctx context.Context) error {
 		case v := <-configWatcher:
 			cluster := v.(state.EventUpdateCluster)
 			d.mu.Lock()
-			d.config.HeartbeatPeriod = time.Duration(cluster.Cluster.Spec.Dispatcher.HeartbeatPeriod)
-			d.nodes.updatePeriod(d.config.HeartbeatPeriod, d.config.HeartbeatEpsilon, d.config.GracePeriodMultiplier)
+			if cluster.Cluster.Spec.Dispatcher.HeartbeatPeriod != nil {
```

Comment:
if it's `nil`, `d.config.HeartbeatPeriod` won't be updated, so the old value (default value, or value from previous config update) of `d.config.HeartbeatPeriod` will be used.